addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1887,6 +1887,28 @@ export class OdooEditor extends EventTarget {
                 restore(); // Make sure to make <br>s visible if needed.
             }
         }
+
+        // we get the `columnsContainer` (.o_text_columns) in case the user added columns and is deleting them 
+        let columnsContainer = null;
```

Comment:
You don't mention `o_text_columns` in the code here , how can you be sure that `row` exist only when we are in columns context ?

Reply:
makes sense.